languages.toml

```diff
@@ -1917,3 +1917,25 @@ roots = []
 [[grammar]]
 name = "xml"
 source = { git = "https://github.com/RenjiSann/tree-sitter-xml", rev = "422528a43630db6dcc1e222d1c5ee3babd559473" }
+
+[[language]]
+name = "wit"
+scope = "source.wit"
+injection-regex = "wit"
+file-types = ["wit"]
+roots = []
+comment-token = "//"
+indent = { tab-width = 2, unit = "  " }
+
+[language.auto-pairs]
+'(' = ')'
+'{' = '}'
+'[' = ']'
+'"' = '"'
+"'" = "'"
+"<" = ">"
```

Comment:
These are used above in XML so that `<` gets closed by `>` but shouldn't be necessary for most other languages. (If not specified, the default pairs are used.) Is `>` or `<` never used as an operator in wit?

Reply:
`<` & `>` in WIT is for template parameters only, e.g: `option<T>`, `result<T, E>`
and in WIT, there is no expression, only type & interface declaration, so I think `<>` could be paired.

Reply:
> Could you add a license to tree-sitter-wit?

new commit update the rev of `tree-sitter-wit` which include a LICENSE file & license field of `package.json` (MIT license)